components/bl-transferList-component/src/helpers/get-toggle-icon.js

```diff
@@ -0,0 +1,15 @@
+import { iconsMap } from './icons';
+
+export const getToggleAllIcon = (isChecked, isIndeterminated) => {
+  if (isIndeterminated === 'undefined') {
```

Comment:
what values can contain the `isIndeterminated`  parameter?

Reply:
It can be boolean or undefined. It is undefined when the user has selected the 'basic' list type.